app/assets/javascripts/app.js

```diff
@@ -1,10 +1,13 @@
 import './libs';
 import ProjectsIndexController from './controllers/projects/IndexController';
+import TagGroupsIndexController from './controllers/tag_groups/IndexController';
 import { start as ProjectsShowController } from './central'
 
 const routes = {
   'projects.index': ProjectsIndexController,
-  'projects.show': ProjectsShowController
+  'projects.show': ProjectsShowController,
+  'tag_groups.new': TagGroupsIndexController,
```

Comment:
Why is it called `TagGroupsIndexController` if it's being used for both `new` and `edit`?